pkg/apis/autoscaling/v1alpha1/metricsproducer_status.go

```diff
@@ -48,6 +48,14 @@ type QueueStatus struct {
 }
 
 type ScheduledCapacityStatus struct {
+	// The current recommendation - the metric the MetricsProducer is emitting
+	CurrentRecommendation *int32 `json:"currentRecommendation,omitempty"`
+
+	// The time the NextRecommendation's schedule will match
+	NextChangeTime *apis.VolatileTime `json:"nextChangeTime,omitempty"`
```

Comment:
NextChangeTime -> NextTime?

Reply:
I agree that it shouldn't be NextChangeTime, after thinking about it. I don't quite agree with NextTime. How about `NextRecommendationTime`?